types/office-js-preview/index.d.ts

```diff
@@ -10559,6 +10559,56 @@ declare namespace Office {
          *                 in the `asyncResult.value` property.
          */
         getInitializationContextAsync(callback: (asyncResult: Office.AsyncResult<string>) => void): void;
+        /**
+         * Gets the Exchange Web Services (EWS) item class of the current appointment.
+         *
+         * @remarks
+         * [Api set: Mailbox preview]
+         *
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **read item**
+         *
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Appointment Organizer
+         *
+         * **Important**:
+         *
+         * - To preview the `getItemClassAsync` method, you must install Outlook on Windows Version 2402 (Build 17317.20000) or later. Then, join the
+         * {@link https://insider.microsoft365.com/join/Windows | Microsoft 365 Insider program} and select the **Beta Channel** option to access Office beta builds.
+         *
+         * - You can create custom classes that extend a default item class. For example, `IPM.Appointment.Contoso`.
```

Comment:
I'm not sure how this example applies to the statement about extending Item. Seems like it's missing context.

Reply:
Coming back after the whole PR, it makes sense, but maybe not as a line hanging off of this API?